docs/src/languages/php/fpm.md

```diff
@@ -83,21 +120,29 @@ As an example, consider the following output:
       6 131072
 ```
 
-This indicates that the majority of requests (4800) used 2048 KB of memory.
-In this case that's likely application caching at work.
-Most requests used up to around 10 MB of memory, while a few used as much as 18 MB and a very few (6 requests) peaked at 131 MB.
-(In this example those are probably cache clears.)
+In the above output:
+
+- The majority of requests (4800) peaked at 2048 KB of memory, 
+  likely thanks to application caching.
+- Most other requests (3206) used up to around 10 MB of memory.
+- 758 requests used more than 10 MB and up to around 18 MB of memory.
+- Only 6 requests peaked at 131 MB of memory, 
+  likely due to cache clears.
 
-A conservative approach would suggest an average request memory of 16 MB should be sufficient.
-A more aggressive stance would suggest 10 MB.
-The more aggressive approach would potentially allow for more concurrent requests at the risk of some requests needing to use swap memory, thus slowing them down.
+In such a situation, you might want to remain cautious and set an average request memory of 16 MB.
+A more aggressive approach might be for you to set an average request memory of 10 MB instead.
+Note that, with the more aggressive approach, there is a risk of allowing more concurrent requests 
+and needing to use swap memory. 
```

Comment:
:pick: To try to for more directness and fewer words
```suggestion
With the more aggressive approach, you a risk of allowing more concurrent requests and needing to use swap memory. 
```

Although the idea of `swap memory` isn't clear to everyone and may be more confusing than helpful. :shrug: